maat-court-data-api/src/test/java/gov/uk/courtdata/dces/controller/FdcContributionsRestControllerTest.java

```diff
@@ -0,0 +1,79 @@
+package gov.uk.courtdata.dces.controller;
+
+import gov.uk.courtdata.dces.response.FdcContributionsResponse;
+import gov.uk.courtdata.dces.service.FdcContributionsService;
+import gov.uk.courtdata.enums.FdcContributionsStatus;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import static org.hamcrest.Matchers.hasSize;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(FdcContributionsRestController.class)
+@AutoConfigureMockMvc(addFilters = false)
+class FdcContributionsRestControllerTest {
```

Comment:
Can you include at a minimum a happy path and unhappy path integration test for the new controller also?